util/sortGroupsBySimilarity.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * @method sortGroupsBySize
```

Comment:
We should update these descriptions at some point.

Reply:
agreed will do it later